pages/searchfilter/[searchitem].js

```diff
@@ -1,109 +1,110 @@
-//         allowClear 
-//         enterButton="Search"
-//         size="large"
-//         enterButton={false}
-//         onSearch={(value) => {
-//           if(value || value.trim()){
-//             getSearchResults(value).then((result) => {
-//               var restoList = result.data
-
-//               if(restoList.length == 0){
-//                 setResultStatus('No results found')
-//               }
-//               else{
-//                 setResultStatus('')
-//               }
-
-//               console.log('restolist:')
-//               console.log(restoList)
-//               setRestaurants(restoList)
-//               router.push('/searchfilter/' + value, undefined, {shallow: true})
-              
-//             })
-//           }
-          
-//         }}
-//       />
-
-//       <Row>
-//         <Col span={6} className={styles.filterLayout}>
-//           <FilterSection></FilterSection>
-//         </Col>
-
-//         <Col span={18} className={styles.cardsLayout}>
-//           <h2>{resultStatus}</h2>
-//           { cards }
+import styles from '../../styles/searchfilter/SearchFilter.module.css'
+import SearchRestoCard from '../../components/searchfilter/SearchRestoCard'
+import FilterSection from '../../components/searchfilter/FilterSection'
+import {Row, Col} from 'antd'
+import Search from 'antd/lib/input/Search'
+
+import { connectToDatabase } from '../../util/mongodb'
+import { useState } from 'react'
+import { useRouter } from 'next/router'
+
+export default function SearchFilter({ results }) {
+  const router = useRouter();
+  const { searchitem } = router.query
+
+  const [restaurants, setRestaurants] = useState(results);
+  const [resultStatus, setResultStatus] = useState(getDefaultResultStatus(restaurants));
+
+  const cards = generateRestaurantCards(restaurants)
+
+  return (
+
+    <div className={styles.pageLayout}>
+
+    <h1>Search Results on "{searchitem}"</h1>
+
+      <Search 
+        className={styles.searchBar} 
+        placeholder="Search Restaurant" 
+        allowClear 
+        size="large"
+        enterButton={false}
+        onSearch={(value) => {
+          if(value && value.trim()){
+            getSearchResults(value).then((result) => {
+              var restoList = result.data
+              if(restoList.length == 0){
+                setResultStatus('No results found')
+              }
+              else{
+                setResultStatus('')
+              }
+              setRestaurants(restoList)
+              router.push('/searchfilter/' + value, undefined, {shallow: true})
+            })
+          }
           
-//         </Col>
-        
-//       </Row>
+        }}
+      />
+
+      <Row>
+        <Col span={6} className={styles.filterLayout}>
+          <FilterSection></FilterSection>
+        </Col>
+        <Col span={18} className={styles.cardsLayout}>
+          <h2>{resultStatus}</h2>
+          { cards }
+        </Col>
```

Comment:
I suggest we leave out the filter component first and make the result cards bigger first as our goal is to make it ready for production. Try commenting out both columns and this column in the meantime.

`        <Col span={24} className={styles.cardsLayout}>
          <h2>{resultStatus}</h2>
          { cards }
        </Col>`